RELEASE_NOTES.md

```diff
@@ -4,7 +4,9 @@
 ### Features
 - Podman has seen an extensive rewrite of its network stack to add support for Netavark, a new tool for configuring container networks, in addition to the existing CNI stack. Netavark will be default on new installations when it is available.
 - The `podman network connect` command now supports three new options, `--ip`, `--ip6`, and `--mac-address`, to specify configuration for the new network that will be attached.
+- The `podman network create` command now allows the `--subnet`, `--gateway`, and `--ip-range` options to be specified multiple times, to allow for the creation of dual-stack IPv4 and IPv6 networks with user-specified subnets.
 - The `--network` option to `podman create`, `podman pod create`, `podman run`, and `podman play kube` can now, when specifying a network name, also specify advanced network options such as `alias`, `ip`, `mac`, and `interface_name`, allowing advanced configuration of networks when creating containers connected to more than one network.
+- Root Podman can now forward ports from IPv6 addresses on the host into containers when using the new Netavark network stack.
```

Comment:
Where does this come from? We still do not have a default ipv6 network so this will not work.

Reply:
If you do have a v6 network created, it will work; previously it wouldn't work at all. I can clarify further?

Reply:
CNI can do this to! I do not understand why you guys think we improved on the ipv6 front. We basically use the same rules as CNI.

Reply:
I'd suggest changing "Root Podman" to "Podman running as root" or "Rootful Podman"

Reply:
Removed